test/fail_compilation/fail21613.d

```diff
@@ -0,0 +1,20 @@
+// https://issues.dlang.org/show_bug.cgi?id=21613
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail21613.d(14): Error: Cannot define both an rvalue constructor and a copy constructor for `struct Test`
+fail_compilation/fail21613.d(19):        Template instance `fail21613.Test.__ctor!()` creates a rvalue constructor for `struct Test`
+---
+*/
+
+struct Test
+{
+    this(ref const typeof(this) rhs){}
+    this()(const typeof(this) rhs){}    // rvalue ctor
+}
+
+void main(){
+    const Test cb;
+    Test b = cb;
+}
```

Comment:
You could add this test to `fail_compilation/test22593.d` because it checks another manifestation of the same error.